recipes/eigen/bld.bat

```diff
@@ -0,0 +1,27 @@
+mkdir build
+cd build
+
+set CMAKE_CONFIG="Release"
+
+if "%PY_VER%" == "3.4" (
+    set GENERATOR=Visual Studio 10 2010
+) else (
+    if "%PY_VER%" == "3.5" (
+        set GENERATOR=Visual Studio 14 2015
+    ) else (
+        set GENERATOR=Visual Studio 9 2008
+    )
+)
+
+if %ARCH% EQU 64 (
+    set GENERATOR=%GENERATOR% Win64
+)
```

Comment:
Do we still need all this generator stuff if we aren't doing testing? Does this affect how things are configured by Eigen?